tests/testthat/test-get_code.R

```diff
@@ -267,6 +267,19 @@ testthat::test_that("detects function usage of the assignment operator", {
 
 # @linksto ---------------------------------------------------------------------------------------------------------
 
+testthat::test_that("get_code does not break if @linksto is with eval in last line", {
```

Comment:
Maybe clarification like this? 
```suggestion
testthat::test_that("get_code does not break if @linksto is put in the last line", {
  # In some cases R parses comment as a separate expression so the comment is not 
  # directly associated with this line of code. This situation occurs when `eval` is in the last
  # line of the code. Other cases are not known but are highly probable.
```

Reply:
thanks @gogonzo